src/frontend/js/modules/http.js

```diff
@@ -53,7 +53,7 @@
 
   .provider('httpConfigurer', function() {
     var restangulars = [];
-    var baseUrl = '';
+    var baseUrl = window.openpaas && window.openpaas.BASE_URL ? window.openpaas.BASE_URL : '';
```

Comment:
You should use the setBaseUrl function below which updates all the restangular instances. Do it from a `angular.run` block somewhere